replication.m

```diff
@@ -51,27 +51,135 @@
 function horizonPersistRegData = runHorizonPersistReg(data)
     forecastColumns = {'CPI1', 'CPI2', 'CPI3', 'CPI4', 'CPI5', 'CPI6'};
     results = struct();
-    
+
     for h = 1:5
-        X = [ones(size(data, 1), 1), data.(forecastColumns{h})];
         Y = data.(forecastColumns{h+1});
-        regress = fitlm(X,Y);
-
-        disp(regress.Coefficients)
+        X = [ones(size(data, 1), 1), data.(forecastColumns{h})];
         
-    
+        beta = (X' * X) \ (X' * Y);
+        
+        residuals = Y - X * beta;
+        
+        stdErr = driscollKraay(X, residuals, data.timeIndex);
+        
+        df = 3;
```

Comment:
df=3 is arbitrary but it comes surprisingly close to replicating the actual interval. I just could not figure out how they calculated their degrees of freedom since the standard I know, num observation - num parameters, yielded values that were consistently about 10% smaller than the intervals shown in the paper.